### PR TITLE
Fix bug: Renew expired token causes an error #24

### PR DESCRIPTION
**Pull request by wp99cp**
Change ```request_headers``` form a global variable to a class attribute.

It seems that the following request changes the ```request_headers```. It adds an ```authId```, since ```request_headers``` was a global variable and ```authId``` isn't a valid header attribute for some request before, this caused bug #24. Changing ```request_headers``` to a class attribute allows multiple token objects in the same runtime-environment.
https://github.com/abertschi/postcard_creator_wrapper/blob/276f099b1633b5c9147ea2ce61d1fbe911dbc780/postcard_creator/token.py#L262-L271

### CODE REVIEW

**Comment by abertschi** [comment]
Looks good, request_headers and headers currently store the same http headers. can you merge them and only use one headers field.

File: postcard_creator/token.py [hunk 1]
```diff
@@ -71,6 +67,11 @@ def __init__(self, _protocol='https://'):
                           'Version/4.0 Chrome/52.0.2743.98 Mobile Safari/537.36',
             'Origin': '{}account.post.ch'.format(self.protocol)
         }
+        self.request_headers = {
+            'User-Agent': 'Mozilla/5.0 (Linux; Android 6.0.1; wv) ' +
+                          'AppleWebKit/537.36 (KHTML, like Gecko) ' +
+                          'Version/4.0 Chrome/52.0.2743.98 Mobile Safari/537.36',
+        }
```
Comment (abertschi) [inline postcard_creator/token.py:74]: merge request_headers and headers

**Review by abertschi — CHANGES_REQUESTED** [review]

File: postcard_creator/token.py (hunk 1 above)
Comment (abertschi) [reply postcard_creator/token.py:74]: alternatively rename them to more meaningful names such as swissid_headers and legacy_headers

Comment (wp99cp) [reply postcard_creator/token.py:74]: Good point, see https://github.com/wp99cp/postcard_creator_wrapper/commit/3693ace8325f875dc07069d340d766ac1875df23.

**Comment by abertschi** [comment]
merge

thx for your contribution. LGTM

**Pull request merged**